nursery/check-for-minimum-number-of-windows-on-screen.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: check for minimum number of windows on screen
+    namespace: anti-analysis/anti-vm/vm-detection
+    authors:
+      - echernofsky@google.com
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Virtualization/Sandbox Evasion::System Checks [T1497.001]
+    references:
+      - https://www.crowdstrike.com/blog/guloader-dissection-reveals-new-anti-analysis-techniques-and-code-injection-redundancy/
+      - https://elis531989.medium.com/dancing-with-shellcodes-cracking-the-latest-version-of-guloader-75083fb15cb4
+    # examples:
+    #   - 
+  features:
+    - and:
+      - api: EnumWindows
+      - instruction:
+        - mnemonic: cmp
+        - number: 0xC
```

Comment:
can you provide a description for what this constant means? and/or a comment explaining how to interpret this.

Reply:
```suggestion
        - number: 0xC = minimum number of windows expected
```